pkg/network/proxyimpl/hybrid/proxy.go

```diff
@@ -140,6 +145,19 @@ func (p *HybridProxier) OnServiceUpdate(oldService, service *corev1.Service) {
 	}
 }
 
+func (p *HybridProxier) CleanupState(svcName types.NamespacedName) {
```

Comment:
this should be a private function, so lowercase this name.

Reply:
also, add a short docblock explaining what this does (handle endpoint and service deletions in any order)

Reply:
> also, add a short docblock explaining what this does (handle endpoint and service deletions in any order)

ack yea good point, I always forget to do this.